jsplot/src/test/java/tech/tablesaw/components/MarkerTest.java

```diff
@@ -16,4 +16,18 @@ public void asJavascript() {
     assertTrue(x.asJavascript().contains("symbol"));
     assertTrue(x.asJavascript().contains("size"));
   }
+
+  @Test
+  public void testCustomPalette() {
+    Marker palette = Marker.builder().colorScale(Marker.Palette.JET).build();
+    assertTrue(palette.asJavascript().contains("colorscale: 'Jet'"), palette.asJavascript());
+    Marker x = Marker.builder().addColorScale(0, 255, 0, 0).addColorScale(1, 0, 255, 0).build();
+    assertTrue(
+        x.asJavascript()
+            .contains(
+                "colorscale: [\n"
+                    + "['0.0', 'rgb(255,0,0)'],\n"
```

Comment:
You might want to look into JSONAssert:

https://www.baeldung.com/jsonassert

https://github.com/skyscreamer/JSONassert